### ISSUE TITLE
cum_to_incr() not working as expected on full_triangle_

### ISSUE THREAD

**Issue by cbalona** [casact/chainladder-python#157]
As of 0.8.3, cum_to_incr() does not actually convert to an incremental triangle.

```python
import chainladder as cl
raa = cl.load_sample('raa')
cl.Chainladder().fit(raa).full_triangle_.cum_to_incr()
```
This will display the cumulative triangle. Running the same under 0.8.2 converts to incremental. Possibly a side effect from #143 ?

**Comment by jbogaardt** [comment]
Definitely is a regression. Sorry about that.  The quick way around it ahead of a fix is:
```python
import chainladder as cl
raa = cl.load_sample('raa')
full = cl.Chainladder().fit(raa).full_triangle_
full.is_cumulative = True
full.cum_to_incr()
```
`cum_to_incr` checks whether the triangle is already incremental before doing any calculations, if it is, then it just hands you the same triangle.

**Comment by jbogaardt** [comment]
Fixed in master, I'll fast track moving to 0.8.4, probably this weekend.
